### PR TITLE
RF | POM > dashboardBreadcrumbs > AT_04.02.004

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://trello.com/c/BtBmDYbg/1605-rf-pom-at0402004-dashboardbreadcrumbs

### CODE REVIEW

**Review by OlgaAbdu — APPROVED** [review]

File: cypress/pageObjects/NewItemPage.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ class NewItemPage {
     getPipelineItem = () => cy.get('ul .org_jenkinsci_plugins_workflow_job_WorkflowJob'); 
     getMultiConfigurationProjectItem = () => cy.get(".hudson_matrix_MatrixProject");
     getOrgFolderItem = () => cy.get('.jenkins_branch_OrganizationFolder');
+    getNewItenHeader = () => cy.get('.header .h3');
```
Comment (S-Lanchik) [inline cypress/pageObjects/NewItemPage.js:16]: I guess you misspelled selector name. Next time don't forget to change it

Comment (SvetlanaKanel) [reply cypress/pageObjects/NewItemPage.js:16]: Thank you, I will

**Review by verafes — APPROVED** [review]

**Comment by S-Lanchik** [comment]
Please change your PR name. For ex., RF | POM > AT_04.02.004 | Verify navigating to the corresponding folder page.
Why did you skip AT_12.05_007?

**Comment by SvetlanaKanel** [comment]
AT_12.05_007 fell and I add an error card in Trello with description

**Comment by SvetlanaKanel** [comment]
According to rename of pr that you asked for, we add the name of the specName and then itName because it will help to find the test if it needs. I have just clarified it with Maria Moroz

**Pull request merged**